src/fbjava-impl/src/main/java/org/firebirdsql/fbjava/impl/ExternalEngine.java

```diff
@@ -581,7 +581,7 @@ void putInMessagePrivileged(final IExternalContext context, final Pointer messag
 					}
 				};
 			}
-		}, new DataTypeReg(Blob.class, "java.sql.Blob"));
+		}, new DataTypeReg(Blob.class, "java.sql.Blob"), new DataTypeReg(org.firebirdsql.jdbc.FirebirdBlob.class, "org.firebirdsql.jdbc.FirebirdBlob"));
```

Comment:
Please import `FirebirdBlob` and use unqualified name in first parameter.

You also need change `README.md`.